docs/docs/context.md

```diff
@@ -1,25 +1,23 @@
 ---
 id: context
 title: Context
-permalink: docs-old/context.html
-prev: advanced-performance.html
+permalink: docs/context.html
 ---
 
-One of React's biggest strengths is that it's easy to track the flow of data through your React components. When you look at a component, you can easily see exactly which props are being passed in which makes your apps easy to reason about.
+With React, it's easy to track the flow of data through your React components. When you look at a component, you can see which props are being passed, which makes your apps easy to reason about.
 
-Occasionally, you want to pass data through the component tree without having to pass the props down manually at every level. React's "context" feature lets you do this.
+In some cases, you want to pass data through the component tree without having to pass the props down manually at every level. There are several popular libraries that help you do this, like [Redux](https://github.com/reactjs/redux) and [MobX](https://github.com/mobxjs/mobx).
+You can also do this directly in React with the powerful "context" API.
 
 > Note:
 >
-> Context is an advanced and experimental feature. The API is likely to change in future releases.
->
-> Most applications will never need to use context. Especially if you are just getting started with React, you likely do not want to use context. Using context will make your code harder to understand because it makes the data flow less clear. It is similar to using global variables to pass state through your application.
+> Context is an advanced and experimental feature. If you are not an experienced React developer, you do not want to use context. If you are getting annoyed by passing props down manually through many levels of a tree, first try using [Redux](https://github.com/reactjs/redux) or [MobX](https://github.com/mobxjs/mobx).
```

Comment:
This makes it seem like these libraries offer an abstraction on top of context. However they don't: libraries you link to are independent of React.

There are **React bindings** for these libraries which happen to use context, but they are called differently (React Redux, mobx-react) and are in separate repos. 

This paragraph also  makes it seem like passing data down is the primary responsibility of these libraries. But both Redux and MobX are much more opinionated about state handling in general.